v1.5/getting-started/mesos/demos/cni/cni/conf/calico-net.conf

```diff
@@ -4,7 +4,7 @@
     "ipam": {
         "type": "calico-ipam"
     },
-    "etcd_authority": "etcd:2379",
+    "etcd_endpoints": "http://etcd:2379",
```

Comment:
Blerge, just noticed you modified v1.5 files. We should probably revert those...